builder.json

```diff
@@ -78,6 +78,27 @@
                         "-DBUILD_TESTING=OFF"
                     ]
                 }
+            },
+            "hosts": {
+                "alpine": {
```

Comment:
suggestion: add a comment explaining why this is necessary
```suggestion
                "_comment": "???????what is going on here?????",
                "alpine": {
```

Reply:
Thanks, I have added a comment.